### PR TITLE
feat(tagging): Include 'location' parameter to track result events.

### PR DESCRIPTION
**Pull request by tajespasarela**
EX-5163

This PR adds the `location` parameter injected by the `LocationProvider` component, to the result tagging event (click).

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Change requires a documentation update

## What is the destination branch of this PR?

- [x] `Main`
- [ ] Other. Specify:

## How has this been tested?

- E2E test
- Manual test

### CODE REVIEW

File: packages/x-components/cypress.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "pluginsFile": "tests/plugins/index.js",
   "supportFile": "tests/support/index.ts",
   "integrationFolder": "tests/e2e",
-  "testFiles": ["cucumber/**/*.features"],
+  "testFiles": ["cucumber/**/*.feature"],
```
Comment (diegopf) [inline packages/x-components/cypress.json:11]: is this change intended? I understood that we should stick to *.features because is faster on headless mode

**Review by diegopf — COMMENTED** [review]
Looks good to me. However I think we should clarify https://github.com/empathyco/x/pull/273/files#r769548556

File: packages/x-components/cypress.json (hunk 1 above)
Comment (tiborux) [reply packages/x-components/cypress.json:11]: it should be a mistake

Comment (tajespasarela) [reply packages/x-components/cypress.json]: Ups, I forgot to change back. 
Fixed now

**Review by tiborux — APPROVED** [review]

**Review by diegopf — APPROVED** [review]
